src/assets/data/tutorials/index.js

```diff
@@ -70,6 +70,20 @@ const data = [
             {
                 link: "/community/members/tanuj-agarwal",
                 name: "Tanuj Agarwal"
+            },
+        ],
+    },
+    {
+        date: "Jan. 28 2021",
+        topic: "Intro to Mesheryctl",
```

Comment:
There is already a topic named "Introduction to mesheryctl". Can we think of a different name? // @leecalcote

Reply:
@Ruth-ikegah fyi - https://layer5io.slack.com/archives/CLVT4TSG4/p1614654270384500

Reply:
@Ruth-ikegah To save your time- "Beginner's guide to contributing to Meshery and mesheryctl" is the decided title.

Reply:
Pushed a commit to sort this now @navendu-pottekkat